.github/workflows/docker.yml

```
@@ -0,0 +1,47 @@
+name: docker
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
```

```suggestion
      - name: Set up QEMU
```

Ditto for the others

---

> Ditto for the others

I will clean up all those and more, this was really just a DRAFT to make sure the idea works